screens/Teams/SpecificTeamScreen.js

```diff
@@ -1,180 +1,230 @@
-import React, { useState, useEffect } from 'react';
-import {
-  Image,
-  StyleSheet,
-  View,
-  ScrollView
-} from 'react-native';
+import React, { useState, useEffect } from "react";
+import { Image, StyleSheet, View, ScrollView } from "react-native";
 
 import KitBackgroundScreen from "../../components/KitBackgroundScreen";
 
-import KitText from '../../components/KitText';
+import KitText from "../../components/KitText";
```

Comment:
Assuming these are all prettier changes

Reply:
One small change that uses the team icon with getIcon instead of just the bunny